src/lib/input/autosize.spec.ts

```diff
@@ -176,6 +177,39 @@ describe('MatTextareaAutosize', () => {
       .toBe(textarea.scrollHeight, 'Expected textarea height to match its scrollHeight');
   });
 
+  it('should not calculate wrong content height due to long placeholders', () => {
+    const fixtureWithPlaceholder = TestBed.createComponent(AutosizeTextareaWithLongPlaceholder);
+    fixtureWithPlaceholder.detectChanges();
+
+    textarea = fixtureWithPlaceholder.nativeElement.querySelector('textarea');
+    autosize = fixtureWithPlaceholder.debugElement.query(
+      By.directive(MatTextareaAutosize)).injector.get<MatTextareaAutosize>(MatTextareaAutosize);
+
+    // To be able to trigger a new calculation of the height with a long placeholder, the textarea
+    // value needs to be changed.
+    textarea.value = '1';
```

Comment:
Make a helper function for this `triggerTextareaResize()`?

Reply:
Yeah that sounds good.